Misc/python.man

```diff
@@ -161,9 +161,9 @@ Further restrictions may be imposed to prevent the user from injecting
 malicious code.
 .TP
 .BI "\-m " module-name
-Searches
+Adds the current directory to the start of 
 .I sys.path
-for the named module and runs the corresponding
+ and runs the corresponding
```

Comment:
The fact that the current directory is added to `sys.path` is not specific to `-m` option, see https://docs.python.org/3/library/sys.html#sys.path. For instance, given the current directory:
```
.
└── pkg
    ├── __init__.py
    └── mod.py
```
any python command will find `pkg.mod` (e.g. `pydoc pkg.mod`).

As the current wording explicitly refers to `sys.path`, I believe this is sufficient.